docs2/site/docs/getting-started/field-middleware.md

```diff
@@ -18,10 +18,10 @@ await schema.ExecuteAsync(_ =>
 });
 ```
 
-You can write a class that has a `Resolve` method:
+And write a class that implements `IFieldMiddleware`:
```

Comment:
Logically it should go first and than section about _You register Field Middleware in `ExecutionOptions`._

Reply:
I agree

Reply:
Take a look now